sema4ai/src/sema4ai_code/robo/collect_actions.py

```diff
@@ -11,6 +11,16 @@
 
 log = get_logger(__name__)
 
+_MetadataType = TypedDict(
+    "_MetadataType",
+    {
+        "actions-spec-version": str,
```

Comment:
The new version has `_` instead of `-` (so, `actions_spec_version` and `data_spec_version`).